packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -690,8 +704,12 @@ export class StripeHelper extends StripeHelperBase {
             enabled: true,
           },
           customer_details: {
-            tax: {
-              ip_address: ipAddress,
+            shipping: {
```

Comment:
So previously this function had ipAddress as a parameter which has been replaced by countryCode and postalCode but they are optional.

for the conditional on line 700 should we check for these values? the automaticTax variable is the config value + whether tax is supported. I am not sure what the behavior would be if we pass automatic_tax without any location info (i am guessing itd error out but will test tomorrow).

Reply:
> should we check for these values
I had considered this too, but if we place the check there, we end up attaching taxrates to the invoice. I'm not super familiar with how that works and if that's desireable.

> i am guessing itd error out
It just returns the invoice without taxes included.

Reply:
I see your comment was on a slightly different diff than the one I was looking at. Yes - it'll error out without some customer details provided. Thus passing tax_exempt: 'none'.